src/main/java/jmgomez/policyapi/exception/ApiExceptionHandler.java

```diff
@@ -0,0 +1,20 @@
+package jmgomez.policyapi.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+@ControllerAdvice
+public class ApiExceptionHandler {
+
+    @ExceptionHandler(value = {ApiRequestException.class})
+    public ResponseEntity<Object> handleApiRequestException(ApiRequestException apiEx) {
+        HttpStatus httpStatus = HttpStatus.BAD_REQUEST;
+        ApiException apiException = new ApiException(apiEx.getMessage(), httpStatus, ZonedDateTime.now(ZoneId.of("Z")));
```

Comment:
I'm not liking the new classes but I think you can user ZoneOffset.UTC instead of Z and it reads better. Can you try?

Reply:
Done, it works